sympy/concrete/summations.py

```diff
@@ -512,6 +512,21 @@ def is_convergent(self):
         except NotImplementedError:
             pass
```

Comment:
```suggestion
            lim_ratio = None
```
It should be defined also when the limit does not exist.

Reply:
I observed the implemented ratio test.  I think this should be:
        ### ---------- Raabe's test -------------- ###
        if `abs(lim_ratio)` == 1:  # ratio test inconclusive

Reply:
Because the `ratio test is inconclusive when abs(lim_ratio) == 1`

Reply:
Done the required changes.